src/bin/set_referrer.rs

```diff
@@ -0,0 +1,30 @@
+/*
+This is an example of setting a refferal code for a wallet.
+*/
+use ethers::signers::LocalWallet;
+
+use hyperliquid_rust_sdk::{BaseUrl, ExchangeClient};
+use log::info;
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+    // Add a testnet key which has deposited some funds on HL testnet
```

Comment:
`// Key was randomly generated for testing and shouldn't be used with any real funds`

Reply:
:thinking: But the wallet needs to have trading enabled to be able to test setting referral code. Anyways, I've changed the comment :saluting_face: